### ISSUE TITLE
Failing to locate the content document for RSS documents, dereferencing null

### ISSUE THREAD

**Issue by eyalroz** [eyalroz/bidimailui#46]
In some cases - specifically, when reading RSS-obtained documents sometimes, I can't cycle directions:
```
  cycleDirectionSettings : function() {
    var messagePane = document.getElementById("messagepane");
    var body = messagePane.contentDocument.body;
    var newForcedDirection;
    switch (body.getAttribute('bidimailui-forced-direction')) {
      case 'ltr':
        newForcedDirection = 'rtl';
        break;
      case 'rtl':
        newForcedDirection = null;
        break;
      default: // should be null
        newForcedDirection = 'ltr';
    }
```
`messagepane` is not null, but the contentDocument is missing/null. So, we fail to get the body.

**Comment by eyalroz** [comment]
Also getting a failure in:
```
  gatherParameters : function() {
    if (!msgWindow) { return [null, null, null]; }
    let domDocument =  msgWindow.messageWindowDocShell.contentViewer.DOMDocument;
```
sometimes.

**Comment by eyalroz** [comment]
This should be resolved by the commits fixing bug #97 for TB 115 and later.
